packages/lune/src/commands/ci.ts

```diff
@@ -18,25 +20,27 @@ WARNING: if it exists, this command WILL delete the to directory silently.
 This is expected to be okay in 99% of cases, but make sure when setting your scripts up that you get it right.
 
 Options:
-  --to: The directory to build the plugin into
   --dev: Disables minification
-  --cfg: Specifies the path to a lune cfg file (default: ./lune.config.js)`,
+  --to: The directory to build the plugin into (default: "./dist")
+  --repoSubDir: The subdir containing plugins in a monorepo (default: "plugins")
+  --cfg: Path to a lune.config.js file to use for all plugins (default: nearest lune.config.js to the plugin)`,
   argSchema: {
     dev: "bool",
     to: "str",
     cfg: "str",
+    repoSubDir: "str",
   },
   async exec(args) {
-    const repoRoot = args[0] ?? ".";
+    const dir = args[0] ?? process.cwd();
 
-    const distDir = resolve(repoRoot, (args.to as string) ?? "dist");
+    const distDir = resolve(dir, (args.to as string) ?? "dist");
     if (existsSync(distDir)) await rm(distDir, { recursive: true });
 
     const timeBefore = hrtime.bigint();
 
-    const cfg = await loadCfg(resolve(repoRoot, (args.cfg as string) ?? "lune.config.js"));
+    const specifiedCfg = await loadCfg(args.cfg as string);
 
-    const pluginsDir = resolve(repoRoot, cfg.repoSubDir ?? "plugins");
+    const pluginsDir = resolve(dir, (args.repoSubDir as string) ?? "plugins");
```

Comment:
NVM, we're removing that cfg option entirely in favour of the flag. Shouldn't break much because I dont think anyone renamed it from plugins in their repo.